nevergrad/optimization/optimizerlib.py

```diff
@@ -2055,6 +2078,49 @@ def _select_optimizer_cls(self) -> base.OptCls:
         return optimClass
 
 
+RecombiningOptimisticNoisyDiscreteOnePlusOne = ParametrizedOnePlusOne(crossover=True, mutation="discrete",
+                                                                      noise_handling="optimistic").set_name(
+    "RecombiningOptimisticNoisyDiscreteOnePlusOne", register=True)
```

Comment:
maybe you don't need register=True ?

Reply:
need it for the pytest in double_o_seven

Reply:
this line is not needed at all, affect the result to optimClass directly, without registering

Reply:
so uncomment it from experimental variant since it seems it is copied from there, and use 
'optimCls = ParametrizedOnePlusOne(crossover=True, mutation="discrete", noise_handling="optimistic")`
when you need it below, this is the actual best way of using it and we need to enforce it.